core/src/stages/connect.cpp

```diff
@@ -219,21 +219,32 @@ SubTrajectoryPtr Connect::merge(const std::vector<robot_trajectory::RobotTraject
                                 const std::vector<planning_scene::PlanningSceneConstPtr>& intermediate_scenes,
                                 const moveit::core::RobotState& state)
 {
+	// set cost
+	double cost = 0;
+	for (const auto& trajectory: sub_trajectories) {
+		for (const double& d: trajectory->getWayPointDurations()) {
```

Comment:
```suggestion
		for (const double& duration: trajectory->getWayPointDurations()) {
```
never use 1 char names